### PR TITLE
Home/About/Project pages test passed

### PR DESCRIPTION
**Pull request by tufanbutuner**
I forgot to do a commit for each individual section as opposed to 1 commit for all features, this will be done next time

### CODE REVIEW

File: lmaooAutomation/Steps/NavBar/NavBarSteps.cs [hunk 1]
```diff
@@ -13,21 +13,30 @@ public class NavBarSteps
         [Then(@"I navigate to the Home Page")]
         public void ThenINavigateToTheHomePage()
         {
-            ScenarioContext.Current.Pending();
+            var driver = (IWebDriver)ScenarioContext.Current["ChromeDriver"];
+            driver.Url = "http://192.168.2.115/Workstation/QA/lmaoo/Home/index.php";
```
Comment (OnikNoor97) [inline lmaooAutomation/Steps/NavBar/NavBarSteps.cs:17]: This is fine

File: lmaooAutomation/Steps/NavBar/NavBarSteps.cs [hunk 2]
```diff
@@ -13,21 +13,30 @@ public class NavBarSteps
...
+            // ScenarioContext.Current.Pending();
         }
 
         // Tufan
         [Then(@"I navigate to the About Page")]
         public void ThenINavigateToTheAboutPage()
         {
-            ScenarioContext.Current.Pending();
+            var driver = (IWebDriver)ScenarioContext.Current["ChromeDriver"];
+            driver.Url = "http://192.168.2.115/Workstation/QA/lmaoo/About/index.php";
```
Comment (OnikNoor97) [inline lmaooAutomation/Steps/NavBar/NavBarSteps.cs:27]: This doesnt test the navigation bar, you are just using the URL. Have a look into how I navigated to the register page.

File: lmaooAutomation/Steps/NavBar/NavBarSteps.cs [hunk 3]
```diff
@@ -13,21 +13,30 @@ public class NavBarSteps
...
+            // ScenarioContext.Current.Pending();
         }
 
         // Tufan
         [Then(@"I navigate to the Project Page")]
         public void ThenINavigateToTheProjectPage()
         {
-            ScenarioContext.Current.Pending();
+            var driver = (IWebDriver)ScenarioContext.Current["ChromeDriver"];
+            driver.Url = "http://192.168.2.115/Workstation/QA/lmaoo/Project/index.php";
```
Comment (OnikNoor97) [inline lmaooAutomation/Steps/NavBar/NavBarSteps.cs:37]: Same as above

File: lmaooAutomation/Steps/NavBar/NavBarSteps.cs (hunk 2 above)
Comment (tufanbutuner) [reply lmaooAutomation/Steps/NavBar/NavBarSteps.cs:27]: I have fixed this

**Pull request merged**